dataeng/jobs/analytics/AnalyticsEmailOptin.groovy

```diff
@@ -133,6 +133,9 @@ class AnalyticsEmailOptin {
                 // This will create python 3.8 venv inside shell script instead of using shiningpanda
                 shell(dslFactory.readFileFromWorkspace('dataeng/resources/setup-platform-venv-py3.sh'))
                 shell(dslFactory.readFileFromWorkspace('dataeng/resources/setup-exporter-email-optin.sh'))
+                shell(dslFactory.readFileFromWorkspace('dataeng/analytics-exporter/email_optin.yaml'))
```

Comment:
I don't think this will work as I believe `shell` is for running a shell command only.

Reply:
Okay, I was just trying to pull in the file, but I think it's already in there, am I correct?